src/common/canvas.c

```diff
@@ -17,6 +17,8 @@ GType gst_imx_canvas_inner_rotation_get_type(void)
 			{ GST_IMX_CANVAS_INNER_ROTATION_270_DEGREES, "Rotate clockwise 270 degrees", "rotate-270" },
 			{ GST_IMX_CANVAS_INNER_ROTATION_HFLIP, "Flip horizontally", "horizontal-flip" },
 			{ GST_IMX_CANVAS_INNER_ROTATION_VFLIP, "Flip vertically", "vertical-flip" },
+			{ GST_IMX_CANVAS_INNER_ROTATION_UL_LR, "Flip upper left/lower right diagonal", "ul-lr-diagonal-flip"},
+		    { GST_IMX_CANVAS_INNER_ROTATION_UR_LL, "Flip upper right/lower left diagonal", "ur-ll-diagonal-flip"},
```

Comment:
Please fix the indenting.

Reply:
UL_LR and UR_LL don't seem to be intuitive names. Why not 90_DEGREES_HFLIP and 90_DEGREES_VFLIP instead?

Reply:
Well, actually it's loosely based on GstVideo rotations [https://gstreamer.freedesktop.org/documentation/video/gstvideo.html#GstVideoOrientationMethod]
```
GST_VIDEO_ORIENTATION_IDENTITY (0) – Identity (no rotation)
GST_VIDEO_ORIENTATION_90R (1) – Rotate clockwise 90 degrees
GST_VIDEO_ORIENTATION_180 (2) – Rotate 180 degrees
GST_VIDEO_ORIENTATION_90L (3) – Rotate counter-clockwise 90 degrees
GST_VIDEO_ORIENTATION_HORIZ (4) – Flip horizontally
GST_VIDEO_ORIENTATION_VERT (5) – Flip vertically
GST_VIDEO_ORIENTATION_UL_LR (6) – Flip across upper left/lower right diagonal
GST_VIDEO_ORIENTATION_UR_LL (7) – Flip across upper right/lower left diagonal
GST_VIDEO_ORIENTATION_AUTO (8) – Select flip method based on image-orientation tag
GST_VIDEO_ORIENTATION_CUSTOM (9) – Current status depends on plugin internal setup
```